docs/guides/integrating-the-feature-hub.md

```diff
@@ -80,15 +77,23 @@ const manager = new FeatureAppManager(registry, {
 });
 ```
 
-## React `FeatureAppLoader`
+## Placing Feature Apps on a Web Page Using React
 
-The React `FeatureAppLoader` allows the integrator to load Feature Apps from a
-remote location.
+An integrator can use the `FeatureAppLoader` or the `FeatureAppContainer` (both
+from the [`@feature-hub/react`][react-api] package) to place Feature Apps on a
+web page that have been [implemented using
+React][implementing-a-feature-app-using-react]. Each of them need the
+`FeatureAppManager` singleton instance.
 
-### `src`
+### React Feature App Loader
 
-A Feature App can be loaded and rendered by defining a `src` which is the URL to
-its module bundle:
+The `FeatureAppLoader` component allows the integrator to load Feature Apps from
+a remote location.
+
+#### `src`
+
+A Feature App can be loaded and placed on a web page by defining a `src` which
```

Comment:
Its place is defined by the location of the FeatureAppLoader. Not by "defining a `src`"...

Reply:
In addition we use the verb "render" below as well...